app/models/article.rb

```diff
@@ -1,4 +1,5 @@
 class Article < ApplicationRecord
   validates_presence_of :title
   validates_presence_of :content
+  belongs_to :user
```

Comment:
**nitpick:** Change the order of these definitions.

This is only because I am terrible, but how would you feel about changing the order of these. It is up to you, but it can help keep your code organised. I found [a style guide][] online if it helps.

[a style guide]: https://rails.rubystyle.guide/#macro-style-methods